include/mercury/kw/APIConfig.h

```diff
@@ -0,0 +1,76 @@
+/**
+ * \file APIConfig.h
+ *
+ * \copyright Copyright (C) 2019 Kano Computing Ltd.
+ *            License: http://www.gnu.org/licenses/gpl-2.0.txt GNU GPL v2
+ *
+ * \brief Configuration file for web services
+ *
+ */
+
+#ifndef INCLUDE_MERCURY_KW_APICONFIG_H_
+#define INCLUDE_MERCURY_KW_APICONFIG_H_
+
+
+#include <yaml-cpp/yaml.h>
+
+#include <memory>
+#include <string>
+#include <vector>
+
+
+namespace Mercury {
+namespace KanoWorld {
+
+
+/**
+ * \class APIConfig
+ * \brief Provides configurations for the API configurations
+ */
+class APIConfig {
```

Comment:
`IAPIConfig`?

Reply:
Definitely want to do this long-term but maybe let's wait until we figure out how we want configs to work across the board here before splitting out what the interface is.